src/QMidiOut.h

```diff
@@ -27,7 +27,7 @@ class QMidiOut
 	void sendEvent(const QMidiEvent& e);
 	void setInstrument(int voice, int instr);
 	void noteOn(int note, int voice, int velocity = 64);
-	void noteOff(int note, int voice);
+	void noteOff(int note, int voice, int velocity = 64);
```

Comment:
The previous value would have been 0, perhaps that should be preserved here?

Reply:
You are right, I have updated this. Sorry for the late response!